github/tests/ReplayData/Issue.testGetCommentsSince.txt

```diff
@@ -0,0 +1,10 @@
+https
+GET
+api.github.com
+None
+/repos/jacquev6/PyGithub/issues/28/comments?since=2012-05-26T13%3A59%3A33Z
```

Comment:
this is the line that was different from the regular "get_comments" test. See the new "since=..." part